cpu/efr32mg1p/Makefile.include

```diff
@@ -0,0 +1,15 @@
+export CPU_ARCH = cortex-m4
+export CPU_FAM = efr32mg1p
+
+# enable memory protection unit
```

Comment:
any reason for enabling memory protection as default for this CPU? Shouldn't this be something that is enabled by the user?

Reply:
I'll remove it from here (I think I found it a good idea to enable it by default).

Reply:
I think it makes sense to remove it. This again a matter of expectation: CPUs should only include what their implementation really needs. Everything else should be up to external modules/applications to define.

Reply:
Done.